webrtc.html

```diff
@@ -8793,57 +8793,49 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
         </li>
         <li>
           <p>Let <var>newState</var> be the new indicated
-          <code><a>RTCIceTransportState</a></code>.
+          <code><a>RTCIceTransportState</a></code>.</p>
         </li>
```

Comment:
Suggestion: Have a step here like the old way of phrasing it: "Let newCandidatePair be a newly created RTCIceCandidatePair representing the indicated pair if one is selected, and null otherwise."

Then in a note explain that this value would be null under such and such circumstances and not null in such and such circumstances. I think that is more readable and you can set [SelectedCandidatePair] to match the underlying pair without conditionals around it. I think having conditionals only increases the risk of not updating the pair sometimes (like going from connected to connected). An explanation is better as a note than as additional algorithm steps.